pkg/datapath/linux/route/route_linux.go

```diff
@@ -52,7 +52,7 @@ func (r *Route) getNetlinkRoute() netlink.Route {
 		Dst:      &r.Prefix,
 		Src:      r.Local,
 		MTU:      r.MTU,
-		Protocol: r.Proto,
+		Protocol: netlink.RouteProtocol(r.Proto),
```

Comment:
FWIW we don't even need this change if we only bump to the commit which includes the fix: https://github.com/vishvananda/netlink/commit/9de6d08565b3a34150e3e42e0528b8e206953ca4

(That said, I'm also fine with bumping to something slightly newer).